docs/references/nextjs/auth.mdx

```diff
@@ -18,19 +18,27 @@ The `auth()` helper does require [Middleware](/docs/references/nextjs/clerk-midd
 
 ### `protect()`
 
-You can use the `protect()` helper to check if a user is authorized to access something, such as a component or a route handler.
-
-`protect()` will return the `Authentication` object if the user is authorized. If the user is not authorized, it will throw a `notFound` Next.js error.
-
 <Callout type="warning">
   `auth().protect()` only works for App Router and is considered experimental.
 </Callout>
 
 You can use the `protect()` helper in two ways:
-- to check if a user is authenticated
-- to check if a user is authorized to access something, such as a component or a route handler
+- to check if a user is authenticated (signed in)
+- to check if a user is authorized (has the correct roles or permissions) to access something, such as a component or a route handler
+
+The `protect()` helper behaves as follows based on user authentication or authorization status:
```

Comment:
I tried - it makes more sense in a flow chart, but I don't really want to put a screenshot.
Another way I thought about breaking it up was like:
```

#### Authentication

If the user is authenticated, `protect()` will return the [`Auth`](/docs/references/nextjs/auth-object) object.

If the user is **not** authenticated:
- If the request is a document request (for example, a user navigates to the page in their browser):
  - `protect()` redirects the user to the sign-in page. This typically occurs when the user navigates directly to the page in their browser.
- If the request is a non-document request (for example, an API request):
  - `protect()` returns a `404` error.

#### Authorization

If the user is authorized, `protect()` will return the [`Auth`](/docs/references/nextjs/auth-object) object.

If the user is **not** authorized,`protect()` will return a `404` error.

```

Reply:
Breaking it up that way is definitely more visually appealing, but the branching logic itself is kind of intimidating.

I added a commit with an example table section to this PR, you can see it in the preview [here](https://clerk.com/docs/pr/1140/references/nextjs/auth#example-table-section). What do you think?

Reply:
ooo I like it - I'm going to change the headings to authenticated vs authorized.

The reason I'm sticking with this language is because it's used all throughout our docs. 
I do agree authn vs authz can be a confusing concept - even I get confused or caught up sometimes LOL but that's why when the terms are first introduced in the doc, I wrote authentication (signed in) and authorization (has the correct roles or permissions) 
<img width="791" alt="Screenshot 2024-06-10 at 14 10 32" src="https://github.com/clerk/clerk-docs/assets/98043211/8af06b5a-d86e-41a9-a3a2-6e85f26dc78c">

I think this is the best way to go since this is the language we use all thru out our docs